packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md

```diff
@@ -46,33 +46,35 @@ The following Event Types are available only in Identity Engine and can't be use
 
-The use of the Classic Engine Reset Factor API for resetting a user's email enrollment is discouraged and considered moot, because email is an auto-enrolling authenticator in Identity Engine. A user's verified `primaryEmail` is always usable as long as the Email authenticator is set to `ACTIVE`, and the user can use it for **recovery only** or for both **authentication and recovery**, depending on the Email authenticator settings.
+Okta discourages the use of the Classic Engine Reset Factor operation for resetting a user's email enrollment. This is because email is an auto-enrolling authenticator in Identity Engine. A user's verified `primaryEmail` is always usable as long as the Email authenticator is set to `ACTIVE`. The user can use it for **recovery only** or for both **authentication and recovery**, depending on the Email authenticator settings.
 
 ***
 
 ### Reset Factor API - question enrollment
 
-**What Changed:** Identity Engine steers away from the notion of separate questions for MFA and recovery. Therefore, the GET `/factors` API now returns the recovery question (Forgot Password Question) in the absence of an MFA Security Question enrollment for the user.
+**What Changed:** Identity Engine steers away from the notion of separate questions for MFA and recovery. Therefore, the GET `/factors` API now returns the recovery question (forgot password question) in the absence of an MFA Security Question enrollment for the user.
 
-In Classic Engine, when a user is using both the Forgot Password Question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, only the Security Question is reset. And then, if the GET `/factors` API is called, the Forgot Password Question isn't returned as a factor. With an upgrade to Identity Engine, after resetting all the factors, when the GET `/factors` API is then called, the Forgot Password Question is returned as a factor in the response.
+In Classic Engine, when a user is using both the forgot password question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, only the Security Question is reset. If the GET `/factors` API is called, the forgot password question isn't returned as a factor.
 
-> **Note:** With Identity Engine, if a user is using both the Forgot Password Question and a Security Question for MFA, and an API call is made to `v1/lifecycle/reset_factors` to reset all the factors for the user, just the Security Question is reset with that call. To reset the Forgot Password Question after that first call, make a second call to `/v1/lifecycle/reset_factors`.
+With an upgrade to Identity Engine, after resetting all the factors, when the GET `/factors` API is then called, the forgot password question is returned as a factor in the response.
```

Comment:
There are a lot of setup phrases here. Would it work to simplify to something like this?

After you upgrade to Identity Engine and reset all factors, calling the GET `/factors` API returns the forgot password question in the response.